sample.regexp

```diff
@@ -0,0 +1,34 @@
+\k<n>     \k'n'                                   [[:^word]]   ASCII-range         Full-range        Backslash
+\k<-n>    \k'-n'                                  ————————————————————————————————————————————————————————————
+\k<name>  \k'name'                                alpha        \p{PosixAlpha}      \p{XPosixAlpha}
+                                                  alnum        \p{PosixAlnum}      \p{XPosixAlnum}
+\k<n+level> \k'n+level'                           ascii        \p{ASCII}
+\k<n-level> \k'n-level'                           blank        \p{PosixBlank}      \p{XPosixBlank}   \h
+                                                                                   \p{HorizSpace}
+\A(?<a>|.|(?:(?<b>.)\o{22}qr\g<a>\k<b>))\z        cntrl        \p{PosixCntrl}      \p{XPosixCntrl}
+\A(?<a>|.|(?:(?<b>.)\o{22}qr\g<a>\k<b+0>))\z      digit        \p{PosixDigit}      \p{XPosixDigit}   \d
+                                                  graph        \p{PosixGraph}      \p{XPosixGraph}
+\h, \H                                            lower        \p{PosixLower}      \p{XPosixLower}
+(?<name>...), (?'name'...)                        print        \p{PosixPrint}      \p{XPosixPrint}
+\k<name>                                          punct        \p{PosixPunct}      \p{XPosixPunct}
+\g<name>, \g<group—num>                                        \p{PerlSpace}       \p{XPerlSpace}    \s
```

Comment:
See above. `\g<group—num>` contains an em dash.

Reply:
Ah, oops — that was not intended. I'll fix shortly.

Reply:
Done.